.idea/runConfigurations/pcr_example_result_file_rt_pcr.xml

```diff
@@ -0,0 +1,23 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="pcr.example_result-file_rt-pcr" type="PythonConfigurationType" factoryName="Python" folderName="pcr">
+    <option name="INTERPRETER_OPTIONS" value="" />
+    <option name="PARENT_ENVS" value="true" />
+    <envs>
+      <env name="PYTHONUNBUFFERED" value="1" />
+      <env name="PYCHARM_HOSTED" value="1" />
+      <env name="PYTHONIOENCODING" value="UTF-8" />
+    </envs>
+    <option name="SDK_HOME" value="" />
+    <option name="WORKING_DIRECTORY" value="$PROJECT_DIR$/clarity-ext-scripts/repos/clarity-ext/clarity_ext" />
+    <option name="IS_MODULE_SDK" value="true" />
+    <option name="ADD_CONTENT_ROOTS" value="false" />
+    <option name="ADD_SOURCE_ROOTS" value="false" />
+    <module name="claritylims" />
+    <EXTENSION ID="PythonCoverageRunConfigurationExtension" enabled="false" sample_coverage="true" runner="coverage.py" />
+    <option name="SCRIPT_NAME" value="$PROJECT_DIR$/clarity-ext-scripts/repos/clarity-ext/clarity_ext/cli.py" />
+    <option name="PARAMETERS" value="--level INFO extension --cache True clarity_ext_scripts.covid.pcr.example_result-file_rt-pcr test" />
```

Comment:
I'm confused why this works. I was 99.999% sure that the name couldn't have a dash in it. It does work but I would rename this to `clarity_ext_scripts.covid.pcr.example_result_file_rt_pcr`, i.e. all underscores.

Reply:
I can do it, I'm in the code now.